geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -405,21 +489,52 @@ app.DrawController.prototype.onFeatureChange_ = function(event) {
 
 
 /**
+ * @param {ol.Feature} feature Feature to encode in the URL.
+ * @private
+ */
+app.DrawController.prototype.saveFeatureInMymaps_ = function(feature) {
+  var map_id = this.appMymaps_.getCurrentMapId();
+  var currentFeature = feature;
+  if (goog.isDefAndNotNull(map_id) && this.appMymaps_.isEditable()) {
+    this.appMymaps_.saveFeature(feature, this.map.getView().getProjection())
+      .then(goog.bind(function(resp) {
+          var featureId = resp['id'];
+          currentFeature.set('id', featureId);
+        }, this));
+  }
+};
+
+
+/**
  * @param {Array.<ol.Feature>} features Features to encode in the URL.
  * @private
  */
 app.DrawController.prototype.encodeFeaturesInUrl_ = function(features) {
-  this.ngeoLocation_.updateParams({
-    'features': this.fhFormat_.writeFeatures(features)
-  });
+  var featuresToEncode = [];
+  for (var i in features) {
+    var source = features[i].get('__source__');
+    if (!goog.isDefAndNotNull(source) ||
+        (goog.isDefAndNotNull(source) && source != 'mymaps')) {
+      featuresToEncode.push(features[i]);
+    }
+  }
```

Comment:
Here you could use `Array.prototype.filter` or `goog.array.filter`. For example:

```js
var featuresToEncode = features.filter(function(feature) {
  return feature.get('__source__') != 'mymaps';
});
```